crates/sui-json-rpc/src/api.rs

```diff
@@ -479,6 +480,20 @@ pub trait EventReadApi {
     ) -> RpcResult<EventPage>;
 }
 
+#[open_rpc(namespace = "sui", tag = "Threshold BLS APIs")]
+#[rpc(server, client, namespace = "sui")]
+pub trait ThresholdBlsApi {
+    /// Sign an a Randomness object with threshold BLS.
```

Comment:
Today the JSONRPC api is how clients interact with a fullnode, there isn't a jsonrpc server running on any validators. How is this API intended to be used? Are clients expected to directly contact the validators for this or is there some other workflow?

Reply:
The plan here is for clients to interact directly with this API. In the first stage (this PR), the result will be computed locally, as we currently use the mocked key generator anyhow. Together with the second PR (the Move changes), it will allow developers to start evaluating the new interfaces for working with Randomness.
In the next stage we'll add to validators the required APIs and then change the JSON RPC code to call that new API for computing the response (i.e., similar to quorum driver, just with tbls signatures).

Reply:
Thanks for the explanation and expected plans for this!